.github/pull_request_template.md

```diff
@@ -57,6 +57,18 @@ Common, optional tasks are included here in case you forgot something important.
 
 [release]: https://docs.github.com/en/github/administering-a-repository/managing-releases-in-a-repository
 
+<!--
+##### Environment Variables
+
+Set custom environment variables for your frontend staging deploy.
+Setup your repo to use this feature by following the instructions located here: https://github.com/customink/frontend-gh-actions
+
+Environment Variables
+```
+TEST_VAR=TEST_VAL
+```
+-->
+
```

Comment:
So people would probably consider it a bug if you left this commented and the action still made a .env file with `TEST_VAR=TEST_VAL`... right?
Because I'm sure that will happen right now.
I need to improve the matching to exclude anything commented by markdown